axum/src/extract/ws.rs

```diff
@@ -122,17 +122,21 @@ use tokio_tungstenite::{
 
 /// Extractor for establishing WebSocket connections.
 ///
-/// Note: This extractor requires the request method to be `GET` so it should
-/// always be used with [`get`](crate::routing::get). Requests with other methods will be
-/// rejected.
+/// For HTTP/1.1 requests, this extractor requires the request method to be `GET`;
+/// in later versions, `CONNECT` is used instead. Thus it should either be used
+/// with [`any`](crate::routing::any), or placed behind
+/// [`on`](crate::routing::on)`(`[`MethodFilter`]`::GET.or(`[`MethodFilter`]`::POST), ...)`.
```

Comment:
I guess we're using `POST` here because we don't support `CONNECT` directly?
Do you think that we should add `CONNECT` as well?

Reply:
No, that’s just a typo on my part…

Reply:
Although I do now reälize that `MethodFilter::CONNECT` doesn’t exist. I pushed a commit to consistently use `any` instead.